src/components/Newsletter/Newsletter.js

```diff
@@ -40,7 +42,8 @@ const Newsletter = () => {
           <DiamondButton
```

Comment:
correct the button styles with how they look on dark mode
![image](https://user-images.githubusercontent.com/17585202/116616633-87e6a300-a902-11eb-852b-13c1f65c7be2.png)